networkx/algorithms/chordal/chordal_alg.py

```diff
@@ -242,6 +244,47 @@ def chordal_graph_treewidth(G):
         max_clique = max(max_clique,len(clique))
     return max_clique - 1
 
+@not_implemented_for('directed','multigraph')
+def chordal_simplicial_vertex(G):
+    """Returns the simplicial vertex of the chordal graph G. 
+    
+    Parameters
+    ----------
+    G : graph
+      A NetworkX graph 
+    
+    Returns
+    -------
+    Simplicial vertex of graphs G is vertex v such that closed neighborhood of v is clique in G.
+    
+    Raises
+    ------
+    NetworkXError
+        The algorithm does not support DiGraph, MultiGraph and MultiDiGraph. 
+        If the input graph is an instance of one of these classes, a
+        NetworkXError is raised.
+        The algorithm can only be applied to chordal graphs. If
+        the input graph is found to be non-chordal, a NetworkXError is raised.
+        
+    Examples
+    --------
+    >>> import networkx as nx
+    >>> e = [(1,2),(1,3),(2,3),(2,4),(3,4),(3,5),(3,6),(4,5),(4,6),(5,6),(7,8)]
+    >>> G = nx.Graph(e)
+    >>> nx.chordal_simplicial_vertex(G)
+    1
+
+    References
+    ----------
+    .. [1] http://www.cse.iitd.ernet.in/~naveen/courses/CSL851/Keshav.pdf
+    """
+    if not is_chordal(G):
+        raise nx.NetworkXError("Input graph is not chordal.")
+
+    for node in G:
+     if _is_complete_graph(G.subgraph(G[node])):
```

Comment:
This is an open neighborhood, not a closed neighborhood. Is that the correct implementation? If so, change the documentation to state this.